python/Dockerfile

```diff
@@ -0,0 +1,43 @@
+FROM socrata/base
+
+RUN apt-get update
+RUN DEBIAN_FRONTEND=noninteractive apt-get -y install python python-dev python-pip python-numpy python-scipy
```

Comment:
hehehehehe That's fair, but then we'll need to split this into a couple of images.